content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Creating Projects using the Web Editor'
+description: 'An article on how a teacher can use the web editor '
+tags: 
+  - article
+author: 'Lenard George Swamy'
+---
+
+## Introduction 
+
+Many of you might have an interesting idea for your project but struggle with documentation and sharing it with your class. In this article you will learn how to use the web editor to document a project. We will illustrate the process with an example project  
+
+***Note: We recommend that you read the article (Getting started with the Web Editor)(/getting-started/getting-started-web-editor) before you proceed with this tutorial.***
+
+## The Essentials 
+Every arduino project has 3 essential elements that will make your teaching effortless. These are
+* Circuit diagrams 
+* Sketch Files 
+* Picture of your project 
+
+### 1. Create a circuit diagram
+Start by thinking about the layout of your circuit. It could be as basic as a pen on paper but more the fidelity the easier it is for your students to understand how to connect the components. 
+
+For this tutorial we created a circuit of the proximity meter using [tinkercad](https://www.tinkercad.com/circuits).
+
+![proximity_meter_circuit.png](assets/classroom_project_circuit_diagram_proximity_meter.png)
+
+### 2. Connect the components 
+Once you have created the components, its good to connect the components to ensure all the connections are right. Sometimes its also good to support your document with photos of your connections. Heres a picture of the circuit. This gives your students a good visual sense of how the circuit looks like in real life.
+
+![proximity_meter_real_components.png](assets/classroom_project_photo_proximity_meter.png)
+
+### 3. Open a new sketch.
+Open the Web Editor by clicking on the ![header_dropdown_icon.png](assets/icon_header_dropdown.png) icon in the header. 
+
+![proximity_meter_real_components.png](assets/classroom_project_webeditor.png)
+
+When your editor is ready, create a new sketch by clicking ![new_sketch_icon.png](assets/icon_new_sketch.png) and rename the sketch to the name of your project. 
+
+![classroom_project_rename_sketch](assets/classroom_project_rename_sketch.png)
+
+### 4. Write your sketch. 
+Write your sketch or copy-paste your sketch from another project. For easy readibility ensure that
+* Adding comments to describe important statements 
+* Assigning good varible names 
+* Inundating the code 
+
+Here's the example sketch of the Proximity Meter project for your reference.  
+
+<div class="sketch-iframe" data-src="https://create.arduino.cc/editor/ArduinoEdu/e833b75d-5989-4049-aec4-a2472303415d/preview?embed&snippet=L73-L77&hideNumbers&height=120#L73-L77"></div>
```

Comment:
This doesn't show up in the preview, we could have to show this as just code instead.

Reply:
Yeah I saw that too I will remove it